### ISSUE TITLE
Remove Bazel 0.4.4 from supported versions [phab:D1513]

### ISSUE THREAD

**Issue by kythe** [google/kythe#1925]
_Created by **schroederc** at 2017-04-12 16:16:28_: 

Bazel 0.4.4 now emits the following error during a build:

```
ERROR: tools/cpp/clang_configure.bzl:777:15: unexpected keyword 'environ' in call to repository_rule(implementation: function, *, attrs: dict or NoneType = None, local: bool = False)
ERROR: package contains errors: kythe/data
ERROR: error loading package '': Extension file 'tools/cpp/clang_configure.bzl' has errors
```

**Comment by kythe** [comment]
_**schroederc** commented inline (context lost during migration) at 2017-04-12 16:18:26_:

Also, the build error happens even before this check somehow... :-(

**Comment by kythe** [comment]
_**fromberger** commented inline (context lost during migration) at 2017-04-12 16:33:43_:

Ah, because the `load` statements are resolved during load, whereas I think this check doesn't happen till analysis. :(

**Comment by kythe** [comment]
_**fromberger** accepted this revision at 2017-04-12 18:52:27_

**Comment by kythe** [comment]
_**schroederc** closed via commit 3336777c4b361fe7351257fae13150024326b1bd at 2017-04-12 18:53:39_
